index.html

```diff
@@ -1854,6 +1854,53 @@ <h2>Default Language</h2>
           (e.g., en, de-AT, gsw-CH, zh-Hans, zh-Hant-HK, sl-nedis).</span>
         </span>
     </section>
+    <section id="sec-identifiers">
+      <h2>Identifiers</h2>
+      <p>
+        Web resources are identified by uniform resource identifiers (URIs).
+        The Thing Description defines the field <code>id</code> for that purpose, which is in the format "anyURI".
```

Comment:
Reference needed for "anyURI" (or the `id` member in TD which in turn has a reference for "anyURI")

Reply:
reference was added.